### ISSUE TITLE
Invalidate sessions when Mage_Log_Model_Adminhtml_System_Config_Source_Loglevel::LOG_LEVEL_NONE

### ISSUE THREAD

**Issue by rvelhote** [OpenMage/magento-lts#1005]
If you have sessions in two (or more) different browsers/devices changing the password on one of them should invalidate all other sessions.

This is Magento's behaviour in the administration area and also for authenticated customers except when the configured log level for `system/log/enable_log` is `Mage_Log_Model_Adminhtml_System_Config_Source_Loglevel::LOG_LEVEL_NONE` (i.e. disabled).

With this setting disabled the authenticated user's `customer_id` [does not get set](https://github.com/OpenMage/magento-lts/blob/684eb1aa27d29bb58dcbaeca9658b816d5fa9788/app/code/core/Mage/Log/Model/Visitor.php#L226) correctly which causes the following [session validation check](https://github.com/OpenMage/magento-lts/blob/684eb1aa27d29bb58dcbaeca9658b816d5fa9788/app/code/core/Mage/Core/Model/Session/Abstract/Varien.php#L537) for the `password_created_at` attribute to not be considered.

The consequence is that sessions in the various browsers are still active even after a password change and will remain active until the cookie expires or they are logged out.

Can anyone confirm this issue when having this setting disabled? Any suggestions on how to fix this issue?

The easiest way would be to move `$this->setData($this->_session->getVisitorData());` to happen before checking `$this->_skipRequestLogging || $this->isModuleIgnored($observer)` however I believe that the sessions invalidation for password changes should not be coupled with the visitor log.

**Comment by andthink** [comment]
We disabled the visitor log for performance reason and incurred into this problem too, I agree with @rvelhote 
the sessions invalidation for password changes should not be coupled with the visitor log.

**Comment by rvelhote** [comment]
@Schrank Can you confirm if this issue is fixed or related with MO-14 and MO-17 patches from MageOne?

**Comment by Schrank** [comment]
@rvelhote Yes the two patches solve the problem, but only if the unique session feature is enabled.
